server/templates/static/homepage.html

```diff
@@ -57,119 +57,207 @@ <h2>{% trans %}What's new{% endtrans %}</h2>
         </li>
       </ul>
     </li>
-
     <li>
-      {# TRANSLATORS: Header for a set of links to the Data Commons Scatter Plot Explorer tool, where users are able to create scatter plots on many topics. Please see this string in action at https://datacommons.org #}
-      <a class="main" href="{{ url_for('tools.scatter') }}">{% trans project="Data Commons" %}Create Scatter Plots{% endtrans %}</a>:
+      <span class="main">Education</span>:
       <ul>
+        <li>
+          <a class="sub" href="{{ url_for('ranking.ranking', stat_var='Amount_EconomicActivity_ExpenditureActivity_EducationExpenditure_Government_AsFractionOf_Amount_EconomicActivity_ExpenditureActivity_Government', place_type='Country') }}">Countries with Highest Government Spending on Education</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=Count_School&pcx=1&svy=Count_Person_Literate&pcy=1&epd=country/IND&epn=India&epts=Country&ept=AdministrativeArea1&qd=0&ld=0&dd=0&ct=1&rg=0">Literacy vs Schools Per Capita in India</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=DropoutRate_Student_Female_PrimarySchool&svy=Count_School_HasFunctionalToilet&pcy=1&epd=country/IND&epn=India&epts=Country&ept=AdministrativeArea1&qd=0&ld=0&dd=0&ct=0&rg=0">Schools with Functional Toilets vs Female Dropout in India</a>
+        </li>
         <li>
           {# TRANSLATORS: Link to a scatter plot comparing bachelor degree attained vs females per capita across counties in California. Please see this string in action at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=Count_Person_Female&svpx=0-8-0&svdx=Count_Person&svnx=Female&pcx=1&svy=Count_Person_EducationalAttainmentBachelorsDegree&svpy=2-0-6&svdy=Count_Person_25OrMoreYears&svny=Bachelors_Degree&pcy=1&epd=geoId%2F06&epn=California&ept=County">{% trans %}Bachelor Degree Attained vs Females per capita for California Counties{% endtrans %}</a>,
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=Count_Person_Female&svpx=0-8-0&svdx=Count_Person&svnx=Female&pcx=1&svy=Count_Person_EducationalAttainmentBachelorsDegree&svpy=2-0-6&svdy=Count_Person_25OrMoreYears&svny=Bachelors_Degree&pcy=1&epd=geoId%2F06&epn=California&ept=County">{% trans %}Bachelor Degree Attained vs Females per capita for California Counties{% endtrans %}</a>
+        </li>
+        <li>
+          {# TRANSLATORS: Link to a map showing the attainment of Bachelor Degrees or higher across counties in Washington. Please see this string in action at https://datacommons.org #}
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=geoId/53&pn=Washington&pt=State&ept=County">{% trans %}Attainment of Bachelor Degree or Higher across Washington Counties{% endtrans %}</a>
```

Comment:
nit: People with at least a Bachelor's degree across ...

Reply:
I'm inclined to leave this string alone (and the next few), since it's been translated already.. or we revisit later?

Reply:
sg